functions/src/environments/environment.ts

```diff
@@ -1,10 +1,58 @@
 
 export const enviroment: Config = {
   githubWebhook: {
-    url: "https://us-central1-pipelinedashboard-dev-cfa68.cloudfunctions.net/responseGitWebhookRepository",
+    url: "https://us-central1-project-pipeline-15611.cloudfunctions.net/responseGitWebhookRepository",
```

Comment:
I need to move this to CI to change depending on the deployment or local build - this is a note for myself 🤓